package-lock.json

```diff
@@ -23541,6 +23532,7 @@
       "resolved": "https://registry.npmjs.org/fsevents/-/fsevents-1.2.13.tgz",
       "integrity": "sha512-oWb1Z6mkHIskLzEJ/XWX0srkpkTQ7vaopMQkyaEIoq0fmtFVxOthb8cCxeT+p3ynTdkk/RZwbgG4brR5BeWECw==",
       "dev": true,
+      "hasInstallScript": true,
```

Comment:
> `hasInstallScript`: A flag to indicate that the package has a preinstall, install, or postinstall script. [package-lock.json | npm Docs](https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json#packages)

Should I use node version `14.x`? `hasInstallScript` is added when use `12.x` version.

Reply:
I am using version 14.15.1. So I think you can upload the version.

Reply:
In my memory, there was no specific version of Node.js and npm environment. So I had opened #568 to specific it.

Reply:
Github workflows is using the `14.x` version. https://github.com/planetarium/9c-launcher/tree/0a0ecd8/.github/workflows

I will update again with `14x` version.